libs/tdb/Makefile

```diff
@@ -45,9 +47,42 @@ define Build/InstallDev
 	$(CP) $(PKG_INSTALL_DIR)/usr/lib/* $(1)/usr/lib/
 endef
 
+CONFIGURE_ARGS = \
+	--target=$(GNU_TARGET_NAME) \
+	--build=$(GNU_HOST_NAME) \
+	--cross-compile \
+	--cross-answers="$(PKG_BUILD_DIR)/cache.txt" \
+	--program-prefix="" \
+	--prefix=$(CONFIGURE_PREFIX) \
+	--exec-prefix=$(CONFIGURE_PREFIX) \
+	--bindir=$(CONFIGURE_PREFIX)/bin \
+	--sbindir=$(CONFIGURE_PREFIX)/sbin \
+	--libexecdir=$(CONFIGURE_PREFIX)/lib \
+	--sysconfdir=/etc \
+	--datadir=$(CONFIGURE_PREFIX)/share \
+	--localstatedir=/var \
+	--mandir=$(CONFIGURE_PREFIX)/man \
+	--infodir=$(CONFIGURE_PREFIX)/info \
+	$(DISABLE_IPV6)
+
+CONFIGURE_ARGS += \
+	--disable-python \
+	--disable-rpath \
+	--disable-rpath-install \
+	--disable-rpath-private-install
+
+define Build/Configure
+	$(CP) ./files/tdb.cache.txt $(PKG_BUILD_DIR)/cache.txt
```

Comment:
Sorry for my mistake, the missing cache.txt file is upload (pushed a new commit just now) just now, please check it again. :-)